app/models/backend/transaction.rb

```diff
@@ -129,31 +70,6 @@ def self.storage
       @@storage ||= Backend::Storage.instance
     end
 
-    def storage_key
-      self.class.storage_key(@service_id, id)
-    end
-
-    def self.storage_key(service_id, id)
-      ::Service.find(service_id).prefix_key "transactions/#{id}"
-    end
-
-    def to_param
-      id && id.to_s
-    end
-
-    def to_xml(options = {})
```

Comment:
Are you confident this is not used from anywhere? I took a look but there are many calls to `to_xml` from many kinds of objects in porta. It's not easy to be sure.

Reply:
Yes, very confident!

The single purpose of this class is to be used in Stats tests. And I think probably this can be refactored further and cleaned up even more... 

This class seems to be a leftover of some very old code (like 14 years ago), when apparently "system" implemented the `/transactions.xml` API which was later moved to the backend (aka apisonator) project.

Some VERY old tests that seem to be relying on this XML representation: https://github.com/3scale/system/blob/f3704d4734c4799cfb479c31b5d12a00366089a1/test/integration/service_transactions_test.rb#L171-L180

So yeah, I'm very confident that we don't need this anymore :upside_down_face:

Reply:
Maybe add this as a comment on top. Would be very helfult to future readers.

Reply:
> Maybe add this as a comment on top. Would be very helfult to future readers.

What exactly? :) 

This is a comment about some deleted code, once it's gone, there will be no questions about it in the future :grimacing: